### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by sagararyal**
Changes in this pull request:
- Upgrade to Next 10.0
- Login for Users
- Manage Projects
- Edit Profile
- Fixes for Public Profile
- Layers in Explore
- add https/http if project url starts without one

@

### CODE REVIEW

File: pages/complete-signup.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Head from 'next/head';
+
+const config = tenantConfig();
+
+export default function UserProfile() {
+
+  return(
+    <>
+    <Head>
+        <title>{`${config.meta.title} - Complete SignUp`}</title>
```
Comment (sagararyal) [inline pages/complete-signup.tsx:14]: translate

File: src/features/user/ManageProjects/components/ProjectCertificates.tsx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+                        : null}
+
+
+                    {errors.certifierName || errors.issueDate || !isDirty || certifierName === '' ? (
+                        <div className={styles.formFieldLarge} style={{ opacity: 0.35 }}>
+                            <div className={styles.fileUploadContainer}>
+                                <AnimatedButton
+                                    className={styles.continueButton}
+                                >
+                                    Upload Certificate
```
Comment (sagararyal) [inline src/features/user/ManageProjects/components/ProjectCertificates.tsx:215]: add translation

File: src/features/user/ManageProjects/components/ProjectCertificates.tsx [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+                            </AnimatedButton>
+                                <p style={{ marginTop: '18px' }}>
+                                    or drag in a pdf
```
Comment (sagararyal) [inline src/features/user/ManageProjects/components/ProjectCertificates.tsx:218]: add translation

File: src/features/user/ManageProjects/components/ProjectMedia.tsx [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+      if (!res.code) {
+        let tempUploadedData = uploadedImages;
+        tempUploadedData[index].description = res.description;
+        setUploadedImages(tempUploadedData);
+        setIsUploadingData(false)
+        setErrorMessage('')
+      } else {
+        if (res.code === 404) {
+          setIsUploadingData(false)
+          setErrorMessage('Project Not Found')
```
Comment (sagararyal) [inline src/features/user/ManageProjects/components/ProjectMedia.tsx:202]: add translation

File: src/features/user/ManageProjects/components/ProjectMedia.tsx [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+            {youtubeURL && !errors.youtubeURL ? (
+              <iframe src={youtubeURL}></iframe>
+            ) : null}
+          </div> */}
+          <div className={styles.formFieldLarge}>
+            <MaterialTextField
+              inputRef={register({
+                pattern: {
+                  value: /^.*(youtu.be\/|v\/|u\/\w\/|embed\/|watch\?v=|\&v=|\?v=)([^#\&\?]*).*/,
+                  message: "Invalid Youtube Video Link"
```
Comment (sagararyal) [inline src/features/user/ManageProjects/components/ProjectMedia.tsx:226]: add translation

File: src/features/user/ManageProjects/components/ProjectMedia.tsx [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+          : null}
+
+        <div className={styles.formField}>
+          <div className={`${styles.formFieldHalf}`}>
+            <AnimatedButton
+              onClick={handleBack}
+              className={styles.secondaryButton}
+            >
+              <BackArrow />
+              <p>Back to basic details</p>
```
Comment (sagararyal) [inline src/features/user/ManageProjects/components/ProjectMedia.tsx:310]: add translation

**Review by sagararyal — COMMENTED** [review]
notes

**Review by Plant-for-the-Planet — APPROVED** [review]

**Pull request merged**